### ISSUE TITLE
[false-positive] wordpress-takeover

### ISSUE THREAD

**Issue by rtcms** [projectdiscovery/nuclei-templates#1133]
**Nuclei version**
```
nuclei -version 
2.3.1
```

**Nuclei template version**

```
cat ~/.nuclei-config.json

{"templates-directory":"nuclei-templates","current-version":"8.1.2","last-checked":"2021-03-23T16:55:24.566266798Z"}

```

**Template ID**

wordpress-takeover

**Commands to Reproduce**

```
nuclei -t nuclei-templates/takeovers/wordpress-takeover.yaml -target https://templateinternal.events.on-running.com
```

**Additional Comments:** According to [can-i-take-over-xyz](https://github.com/EdOverflow/can-i-take-over-xyz) we have to also add ***.wordpress.com?** to **words** to avoid false positive's.

**Comment by ehsandeep** [comment]
thank you @rtcms for reporting this, should be fixed with https://github.com/projectdiscovery/nuclei-templates/commit/83bab07dd6b31ab38d1b754bfad1cc1ad762c549 also make sure to update to the [latest version of nuclei](https://twitter.com/pdnuclei/status/1374000016990171139).

**Comment by geeknik** [comment]
Better matcher fix here: #1134 👍🏻

**Comment by rtcms** [comment]
@ehsandeep Thank You for quickly solving this issue :)
